src/notifications/mod.rs

```diff
@@ -30,10 +31,14 @@ fn fetch_webhook_url() -> Result<String, VarError> {
 }
 
 fn is_webhook_enabled() -> bool {
-  match fetch_webhook_url() {
-    Ok(url) => !url.is_empty() && Url::parse(url.as_str()).is_ok(),
-    Err(_) => false,
+  let url = fetch_var(WEBHOOK_URL, "");
+  if !url.is_empty() {
+    debug!("Webhook Url found!: {}", url);
+    let is_valid = Url::parse(url.as_str()).is_ok();
+    debug!("Webhook is valid: {}", is_valid);
+    return is_valid;
```

Comment:
Can we `warn!` on the webhook not being a valid url at this point? If it gets here then they tried to pass in _something_, so it would be good to let them know that it wasn't a valid url